input/pagecontent/index.md

```diff
@@ -448,14 +419,15 @@ In this section, we will walk through a realistic example of FSH, line by line.
 * Line 6 gives an id for this profile. The id is used to create the globally unique URL for the profile. The URL is composed of the IG’s canonical URL, the instance type (always `StructureDefinition` for profiles), and the profile’s id.
```

Comment:
This is as close as GH will let me comment, but in the first bullet, there is a typo.  It should be "Lines 1 and 2 define", not "Lines 1 and 2 defines".

Reply:
fixed